back-end/api/quickstart/tests/helper_test.py

```diff
@@ -55,3 +56,9 @@ def get_test_partial_post_fields(i=1, author="testauthorid"):
     'categories': '["Testing"]',
     'commentLink': 'link to comments'
   }
+
+def get_test_inbox_fields():
+  return {
+    'author': 'testAuthor',
+    'items': '[{"type":"post"}, {"type":"comment"}]'
```

Comment:
missing likes?

Reply:
since it's a JSON field it can take in anything, so no need to have likes here